demos/core-us-investment-style/demo.html

```diff
@@ -18,13 +19,13 @@
         </style>
     </head>
     <body>
-        <h1>XRay US Connector</h1>
+        <h1>XRay US Connector Investment Style</h1>
```

Comment:
Let's follow the convention in the title and here too - use the same what is in the menu on the left.